scenario_player/utils/legacy.py

```diff
@@ -149,15 +148,16 @@ def reclaim_eth(
 
     txs = []
     reclaim_amount = 0
+    reclaim_tx_cost = web3.eth.gasPrice * VALUE_TX_GAS_COST
```

Comment:
`web3.eth.gasPrice` is a property that hides a RPC call ... that means this value may change just a few lines below when the transaction is actually sent. Could you please reuse the same value to avoid incoherences? (the gas price could go slightly up and the cost of the transaction too, so the transaction would fail even the check is in-place)

Otherwise LGMT